docs/tools/services/relayer-developer.md

```diff
@@ -11,154 +11,187 @@ This feature is currently in private beta. If you are interested, please [contac
+    Relayer->>LUKSO Network: Deploy UP Contract<br>with deterministic address
+    Relayer->>LUKSO Network: Execute postDeployment setup
+    LUKSO Network-->>Relayer: Deployment Success
+    Relayer-->>App: Return UP Address
+```
 
+This method allows for deterministic deployment across different chains.
+
+```javascript
+async function deployUniversalProfileWithSalt() {
+  const apiKey = 'your-api-key';
+  const url = 'https://relayer-api.testnet.lukso.network/api/universal-profile';
+
+  const salt = ethers.utils.randomBytes(32);
+  const lsp6Controllers = ['0x9d9b6B38049263d3bCE80fcA3314d9CbF00C9E9D'];
+  const lsp3Profile = '0x6f357c6a...'; // Encoded LSP3 Profile Data
+
+  const postDeploymentCallData = generatePostDeploymentCallData(
+    lsp6Controllers,
+    lsp3Profile,
+  );
+
+  const data = {
+    salt,
+    postDeploymentCallData,
+  };
+
+  try {
+    const response = await fetch(url, {
+      method: 'POST',
+      headers: {
+        'X-API-KEY': apiKey,
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(data),
+    });
 
-// final result (to be stored on chain)
-const VerfiableURI =  verfiableUriIdentifier + verificationMethod.substring(2) + verificationDatalength.substring(2) + verificationData.substring(2) + url.substring(2)
-                      ^                        ^                                 ^                                     ^                               ^
-                      0000                     6f357c6a                          0020                                  820464ddfac1be...               696670733a2f2...
+    if (!response.ok) {
+      throw new Error(`HTTP error! status: ${response.status}`);
+    }
 
-// structure of the VerifiableURI
-0x0000 + 6f357c6a +       0020 +                    820464ddfac1bec070cc14a8daf04129871d458f2ca94368aae8391311af6361 + 696670733a2f2f516d597231564a4c776572673670456f73636468564775676f3339706136727963455a4c6a7452504466573834554178
-  ^      ^                ^                         ^                                                                  ^
-  0000   keccak256(utf8)  verificationDatalength    verificationData                                                   encoded URL
+    const result = await response.json();
+    console.log('Universal Profile deployed:', result);
+  } catch (error) {
+    console.error('Error deploying Universal Profile:', error.message);
+  }
+}
 
-// example value
-0x00006f357c6a0020820464ddfac1bec070cc14a8daf04129871d458f2ca94368aae8391311af6361696670733a2f2f516d597231564a4c776572673670456f73636468564775676f3339706136727963455a4c6a7452504466573834554178
+deployUniversalProfileWithSalt();
 ```
 
-ℹ️ More info on [`VerifiableURI`](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-2-ERC725YJSONSchema.md#verifiableuri)
+### 2. Register Existing Universal Profiles
 
-</details>
+You can register existing Universal Profiles with the Relayer to enable gasless transactions for your users.
```

Comment:
Suggest also mentioning that the user will have a certain quota amount (20 Millions per month)